lang/ja.json

```diff
@@ -12,15 +12,15 @@
   "Non-Fullscreen": "フルスクリーン以外",
   "Mute": "ミュート",
   "Unmute": "ミュート解除",
-  "Playback Rate": "再生レート",
-  "Subtitles": "サブタイトル",
-  "subtitles off": "サブタイトル オフ",
-  "Captions": "キャプション",
-  "captions off": "キャプション オフ",
+  "Playback Rate": "再生速度",
+  "Subtitles": "吹き替え・字幕１",
+  "subtitles off": "吹き替え・字幕１ オフ",
+  "Captions": "表題・説明・字幕２",
+  "captions off": "表題・説明・字幕２ オフ",
   "Chapters": "チャプター",
-  "You aborted the media playback": "動画再生を中止しました",
-  "A network error caused the media download to fail part-way.": "ネットワーク エラーにより動画のダウンロードが途中で失敗しました",
-  "The media could not be loaded, either because the server or network failed or because the format is not supported.": "サーバーまたはネットワークのエラー、またはフォーマットがサポートされていないため、動画をロードできませんでした",
-  "The media playback was aborted due to a corruption problem or because the media used features your browser did not support.": "破損の問題、またはお使いのブラウザがサポートしていない機能が動画に使用されていたため、動画の再生が中止されました",
+  "You aborted the media playback": "ユーザーにより動画再生を中断しました",
```

Comment:
these set of changes don't seem necessary. They were created via a translation service and seem correct.

Reply:
If there are two "or", it will basically be unnatural Japanese